issue using this library

I wanted to use Photo Sphere Viewer in my project. So I ran

`npm i photo-sphere-viewer`
It appears to have downloaded the modules.

Then inside my project I did:

`import PhotoSphereViewer from 'photo-sphere-viewer/dist/photo-sphere-viewer';`
But I get error:

> Failed to compile.
> 
> Error in ./~/photo-sphere-viewer/dist/photo-sphere-viewer.js
> Module not found: 'D.js' in /home/ghy/WebstormProjects/mia-map/node_modules/photo-sphere-viewer/dist
> 
>  @ ./~/photo-sphere-viewer/dist/photo-sphere-viewer.js 9:4-55

Can anyone help me spot what is wrong?

PS. I think following line is causing issue (inside photo-sphere-viewer.js):

  ```
if (typeof define === 'function' && define.amd) {
    define(['three', 'D.js', 'uevent', 'doT'], factory);
  }
```
but I am even surprised why it enters inside this if as I didn't know I had require.js installed.

PPS. I have a react application created by create-react-app

Hello @psyke089. 
Could you show how you solved this issue please ?

Cause I'm having trouble using this package with React and npm.
I'm having : 
`Module not found: "~\dev\test-360-component\node_modules\doT\doT.js" does not match the corresponding path on disk "dot"` in the terminal. 
And yet at line 12 in photo-sphere-viewer.js : 
`module.exports = factory(require('three'), require('d.js'), require('uevent'), require('dot'));`

--- a/src/js/.wrapper.js
+++ b/src/js/.wrapper.js
@@ -1,9 +1,9 @@
 (function(root, factory) {
   if (typeof define === 'function' && define.amd) {
-    define(['three', 'D.js', 'uevent', 'doT'], factory);
+    define(['three', 'd.js', 'uevent', 'dot/doT'], factory);
   }
   else if (typeof module === 'object' && module.exports) {
-    module.exports = factory(require('three'), require('d.js'), require('uevent'), require('dot'));
+    module.exports = factory(require('three'), require('d.js'), require('uevent'), require('dot/doT'));
   }
   else {
     root.PhotoSphereViewer = factory(root.THREE, root.D, root.uEvent, root.doT);

